features/steps/ccx_inference_service.py

```diff
@@ -31,3 +32,26 @@ def start_ccx_inference_service(context, port):
     assert popen is not None
     time.sleep(0.5)
     context.add_cleanup(popen.terminate)
+
+
+@given("The mock CCX Inference Service is running on port {port:d}")
+def start_ccx_inference_mock_service(context, port):
+    """Run ccx-inference-service for a test and prepare its stop."""
+    params = ["uvicorn", "mocks.inference-service.inference_service:app", "--port", str(port)]
+
+    f = open(f"logs/ccx-upgrades-inference/{context.scenario}.log", "w")
+    popen = subprocess.Popen(params, stdout=f, stderr=f)
+    assert popen is not None
+    time.sleep(0.5)
```

Comment:
dtto

Reply:
+ prob. worth to refactor, as it is the same code as above

Reply:
definitely. I'll merge this PR as-is so I can verify  https://gitlab.cee.redhat.com/ccx/ccx-upgrades-data-eng/-/merge_requests/43/ against the BDDs. But I'm already working on the changes you mentioned.